src/game/inventory/inventory.c

```diff
@@ -521,7 +521,7 @@ static int32_t Inv_ConstructAndDisplay(int inv_mode)
             }
 
             if (start_level != -1 || start_demo
-                || (g_InputDB.option && g_InvMode != INV_TITLE_MODE)) {
+                || ((g_InputDB.option || g_InputDB.roll || g_InputDB.draw) && g_InvMode != INV_TITLE_MODE)) {
```

Comment:
Shouldn't we replace all these checks with just a single `g_InputDB.menu_back`?

Reply:
Doh, yes of course